### PR TITLE
New package: musescore

### PR DESCRIPTION
**Pull request by liwakura**
On behalf of traeak @ IRC.

### CODE REVIEW

File: srcpkgs/MuseScore/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'MuseScore'
+pkgname=MuseScore
+version=2.0.3
+revision=1
+hostmakedepends="cmake doxygen qt5-script qt5-tools qt5-qmake qt-qmake gtk-update-icon-cache pkg-config"
+makedepends="pulseaudio-devel lame-devel portaudio-devel qt5-devel qt5-svg-devel qt5-webkit-devel shared-mime-info libsndfile-devel alsa-lib-devel jack-devel qt5-declarative-devel qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-tds qt5-xmlpatterns-devel qt5-tools-devel"
```
Comment (Vaelatern) [inline srcpkgs/MuseScore/template]: Please line split this so it can fit on a screen for easier viewing.

File: srcpkgs/MuseScore/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz"
+checksum=66c613f27fcc07252e856a51ed1162a9921f3d11ac130260763e2178b7098b01
+
+do_build() {
+	make PREFIX='/usr' release
+}
+
+do_install() {
+	cd build.release
+	make DESTDIR="${masterdir}/destdir/${pkgname}-${version}" install
```
Comment (xtraeme) [inline srcpkgs/MuseScore/template]: This is completely wrong... should be `make DESTDIR=${DESTDIR} install`

File: srcpkgs/MuseScore/template (hunk 1 above)
Comment (xtraeme) [inline srcpkgs/MuseScore/template]: Lots of redundant dependencies...

File: srcpkgs/MuseScore/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Create, play and print beautiful sheet music"
+maintainer="Brian Olsen <bnolsen@comcast.net>"
+license="GPL-2"
+homepage="https://musescore.org/"
+distfiles="https://github.com/musescore/MuseScore/archive/v${version}.tar.gz"
+checksum=66c613f27fcc07252e856a51ed1162a9921f3d11ac130260763e2178b7098b01
+
+do_build() {
+	make PREFIX='/usr' release
+}
```
Comment (Vaelatern) [inline srcpkgs/MuseScore/template]: These functions should not exist. They should probably be replaced with a `build_style`, perhaps `gnu-makefile`, and set `make_build_args="release"`. That should clear up most of the need for these functions.

File: srcpkgs/MuseScore/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'MuseScore'
+pkgname=MuseScore
+version=2.0.3
+revision=1
+hostmakedepends="cmake doxygen qt5-script qt5-tools qt5-qmake qt-qmake gtk-update-icon-cache pkg-config"
```
Comment (xtraeme) [inline srcpkgs/MuseScore/template]: Lots of redundant dependencies...

**Comment by ebfe** [comment]
I'd vote for naming this `musescore` instead of `MuseScore`. Mixed case package names are a bit annoying, but maybe that's just me.

**Comment by xtraeme** [comment]
I'll handle this pull request.

**Pull request closed without merging**